RELEASE/scripts/sl_ascend/sl_batpath.ash

```diff
@@ -355,6 +355,45 @@ boolean bat_shouldEnsorcel(monster m)
 	return false;
 }
 
+int available_total(boolean [item] items)
+{
+	int total = 0;
+	foreach it in items
+	{
+		total += available_amount(it);
```

Comment:
Oh, also, please use `item_amount` instead of `available_amount`. sl_ascend intentionally avoids touching the closet, so people have the option to hide items from it if they want to for some sort of reason.